tutorials/intro/plot_50_configure_mne.py

```diff
@@ -6,7 +6,8 @@
 ======================
 
 This tutorial covers how to configure MNE-Python to suit your local system and
-your analysis preferences.
+your analysis preferences. It also lets you demo things on Discord, for the
+code sprint.
```

Comment:
I'm not sure this is worth mentioning on the website, it's more of a maintainer detail...

Reply:
I know, this is just for demo purposes.  I plan to close it without merging.